lib/index.js

```diff
@@ -12,6 +13,7 @@ module.exports = {
         "no-empty-pattern": "off",
         "playwright/missing-playwright-await": "error",
         "playwright/no-page-pause": "warn",
+        "playwright/no-element-handle": "error",
```

Comment:
I think warning would be more appropriate if we are going to make this part of the default set of rules.

Reply:
But it's really not an error.  It's not recommended, that's for sure, but it won't break your code to use element handles.

Reply:
I see. I'm thinking in another way. For me, it's more like `no-var` eslint rule. Hence, it's not an error to use `var` but the rule is aimed at discouraging the use of `var` and encouraging the use of `const` or `let` instead. As in our case, the `no-element-handle` rule is aimed at discouraging the use of `page.$` and encouraging the use of `page.locator` instead.

Reply:
We have `no-page-pause` as a warning as well, which could be considered another "error" so I'd like to remain consistent with that as I believe setting this to error is not only to heavy handed, but also would be a breaking change.  With a warning we can introduce this and in the future we could possibly increase the level to error.

Reply:
Okay. I've just changed it to `warn`